rinad/configure.ac

```diff
@@ -169,7 +169,7 @@ AS_IF([test "$build_bindings_java" = "yes"],[
 AM_CONDITIONAL(BUILD_BINDINGS_JAVA, [test $build_bindings_java = yes])
 
 AC_PATH_PROG([MVN],[mvn],[])
-AS_IF([test x"$MVN" != x""],[
+AS_IF([test x"$MVN" = x""],[
```

Comment:
Hi Micheal, good to know! In this case, can you change librina/configure.ac to use '=' in place of '=='?
Just to use the same syntax.

Thanks,
  VIncenzo